wire-ios/Wire-iOS/Resources/Configuration/Version.xcconfig

```diff
@@ -16,5 +16,5 @@
 // along with this program. If not, see http://www.gnu.org/licenses/.
 //
 
-WIRE_SHORT_VERSION = 4.11.0
+WIRE_SHORT_VERSION = 4.11.1
```

Comment:
suggestion: when cherry-picking this will be a conflict, I'd suggest to commit this directly to release/cycle-4.11 as a separate commit

Reply:
reverted